### PR TITLE
[FLINK-34740][cdc][doc] copy back Chinese connector docs and update some links.

### PR DESCRIPTION
**Pull request by lvyanquan**
Docs are copied from https://github.com/apache/flink-cdc/tree/release-2.4/docs/content/connectors and https://github.com/apache/flink-cdc/tree/release-2.4/docs/content/%E5%BF%AB%E9%80%9F%E4%B8%8A%E6%89%8B,
and update some links.

### CODE REVIEW

**Comment by lvyanquan** [comment]
@leonardBang @PatrickRen @loserwang1024 PTAL.

File: docs/content.zh/docs/connectors/legacy-flink-cdc-sources/tutorials/mysql-postgres-tutorial.md [hunk 1]
```diff
@@ -24,26 +24,23 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Streaming ETL for MySQL and PostgreSQL with Flink CDC
+# 基于 Flink CDC 构建 MySQL 和 Postgres 的 Streaming ETL
```
Comment (leonardBang) [inline docs/content.zh/docs/connectors/legacy-flink-cdc-sources/tutorials/mysql-postgres-tutorial.md:27]: Could we use shorter title name here?

File: docs/content.zh/docs/connectors/legacy-flink-cdc-sources/tutorials/build-real-time-data-lake-tutorial.md [hunk 2]
```diff
@@ -26,130 +26,117 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Using Flink CDC to synchronize data from MySQL sharding tables and build real-time data lake
+# 基于 Flink CDC 同步 MySQL 分库分表构建实时数据湖
```
Comment (leonardBang) [inline docs/content.zh/docs/connectors/legacy-flink-cdc-sources/tutorials/build-real-time-data-lake-tutorial.md:29]: Could we use shorter title for better display ?

**Review by leonardBang — COMMENTED** [review]
Thanks @lvyanquan for the contribution, I left some minor comments

**Comment by lvyanquan** [comment]
Thanks @leonardBang  for suggestion, reduced the page title to match the document title.

File: docs/content/docs/connectors/legacy-flink-cdc-sources/overview.md [hunk 3]
```diff
@@ -24,12 +24,12 @@ specific language governing permissions and limitations
...
 
-# Legacy CDC sources for Apache Flink
+# Legacy CDC sources
 
-Flink CDC sources is a set of source connectors for <a href="https://flink.apache.org/">Apache Flink<sup>®</sup></a>, ingesting changes from different databases using change data capture (CDC).
+Legacy CDC sources is a set of source connectors for <a href="https://flink.apache.org/">Apache Flink<sup>®</sup></a>, ingesting changes from different databases using change data capture (CDC).
 Some CDC sources integrate Debezium as the engine to capture data changes. So it can fully leverage the ability of Debezium. See more about what is [Debezium](https://github.com/debezium/debezium).
 
-You can also read [tutorials]({{< ref "docs/connectors/legacy-flink-cdc-sources/tutorials/mysql-postgres-tutorial" >}}) about how to use these sources.
+You can also read [tutorials]({{< ref "docs/connectors/legacy-flink-cdc-sources/tutorials/build-streaming-etl-tutorial" >}}) about how to use these sources.
```
Comment (leonardBang) [inline docs/content/docs/connectors/legacy-flink-cdc-sources/overview.md:32]: Could you check other references for these changed links?

Comment (lvyanquan) [reply docs/content/docs/connectors/legacy-flink-cdc-sources/overview.md:32]: I've checked again and found no other links of this page.

**Review by leonardBang — APPROVED** [review]
Thanks @lvyanquan for the update, LGTM

**Pull request merged**